rollup.config.ts

```diff
@@ -302,18 +302,18 @@ function esm({
 }: Options): RollupOptions {
   const bundleOutput: OutputOptions = {
     format: 'esm',
-    file: `${packageDir}/build/lib/${outputFile}.esm.js`,
```

Comment:
All build artifacts were placed in one directory due to `exports` and `typescript` interop.
There were some problems loading types with separate folders setup.